src/main/java/seedu/address/model/person/IsDone.java

```diff
@@ -2,27 +2,37 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.Locale;
+
 /**
  * Represents whether a Person is called in the address book.
  */
 public class IsDone {
 
     public static final String MESSAGE_CONSTRAINTS = "Done has to either be TRUE or FALSE";
-    public final boolean value;
+    public final String value;
```

Comment:
Personally i feel that the value should be kept as a boolean. So can minimise the need for doing isDone.value.equals("True"/"False") whenever we need the value. I think u can just check if whether isDone == "True"/"False" inside the constructor and just set the boolean value accordingly.

Reply:
sure i can make it that way. Not a problem